### ISSUE TITLE
tei tests failing for pandoc-1.17.0.2

### ISSUE THREAD

**Issue by juhp** [jgm/pandoc#2811]
```
:
    tei:
      basic: [Failed]
ERROR: writer.tei: openBinaryFile: does not exist (No such file or directory)
      tables: [Failed]
ERROR: tables.tei: openBinaryFile: does not exist (No such file or directory)
:
```

**Comment by jkr** [comment]
Can't reproduce, for what it's worth.

**Comment by mb21** [comment]
did you do `git submodule update --init` ?

**Comment by jgm** [comment]
Fixed in 1.17.0.3 release.
